### DISCUSSION TITLE
LAN Port Status

### DISCUSSION THREAD

**Discussion by fookyuanloh** [Vaskivskyi/ha-asusrouter#579] [General]
Prior to the update, the status for the LAN ports can be obtained via the attributes - although not as detailed as now.
Since it is now converted into the dict format, how can I access the data?

Pardon me, not a trained programmer - unsure of how the variables can be accessed, but somehow realized the data attribute now is called "List" and based on an array (index) form.

Appreciate some help/guidance on how to display the port status on flex-table-card

**Comment by fookyuanloh** [comment]
any idea?

**Comment by Vaskivskyi** [comment]
Sorry for the late reply. Unfortunately, lately, I didn't have the possibility to check GitHub and answer issues and questions.

Please, allow me some time to find a good suggestion for your use case.

Alternatively, you can check (or ask) in our Community forum discussion, where other users are suggesting their YAML examples to be used with the integration. [Link](https://community.home-assistant.io/t/custom-component-asusrouter-integration/416111?u=vaskivskyi)
